spec/EntitiesSpec.xml

```diff
@@ -405,7 +405,23 @@
   HATCH
 
   -->
-  <!-- TODO MinVersion="R14" -->
+  <Entity Name="Hatch" SubclassMarker="AcDbHatch" TypeString="HATCH" MinVersion="R14" GenerateWriterFunction="false">
+    <Field Name="elevation" Code="30" Type="f64" DefaultValue="0.0" DisableWritingDefault="true" />
```

Comment:
To account for the `10` and `20` codes, this will need to be

``` xml
<Field Name="elevation" Code="10" Type="Point" DefaultValue="Point::origin()" CodeOverrides="10,20,30" />
```

(although with a custom reader/writer function, `CodeOverrides` probably isn't necessary, but it's nice to have to see what might appear in the file)

This means the reader function in `entity.rs` won't be able to assume that all `10` and `20` codes are seed points; `10`/`20` should be used as elevation values until we've read code `98` indicating we're reading the seed point values.